.github/workflows/workflow.yml

```diff
@@ -14,30 +14,29 @@ jobs:
       fail-fast: false
       matrix:
         python-version:
-          - '3.6'
           - '3.7'
           - '3.8'
           - '3.9'
         os:
-          - ubuntu-20.04
+          - ubuntu-22.04
         include:
           - python-version: '3.7'
-            os: macos-latest
+            os: macos-13
           - python-version: '3.7'
             os: windows-latest
```

Comment:
Might be worth pinning this to `windows-2022`, as `windows-latest` will be updated in a few days. See https://github.com/actions/runner-images/issues/12677.